core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-announcements/dot-toolbar-announcements.component.ts

```diff
@@ -31,7 +31,7 @@ import { DotMessagePipe } from '@dotcms/ui';
     templateUrl: './dot-toolbar-announcements.component.html',
     styleUrls: ['./dot-toolbar-announcements.component.scss'],
     standalone: true,
-    imports: [NgForOf, NgClass, DotMessagePipe, RouterLink, CommonModule, OverlayPanelModule],
+    imports: [NgClass, DotMessagePipe, RouterLink, CommonModule, OverlayPanelModule],
```

Comment:
Just asking. This component why is it not OnPush?

Reply:
This change is not covered as part of this card.